src/Test.UnitTests.Sarif/Visitors/SortingVisitorTests.cs

```diff
@@ -0,0 +1,306 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Visitors;
+using Microsoft.CodeAnalysis.Test.Utilities.Sarif;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Visitors
+{
+    public class SortingVisitorTests
+    {
+        private readonly Random random;
+
+        public SortingVisitorTests(ITestOutputHelper outputHelper)
+        {
+            random = RandomSarifLogGenerator.GenerateRandomAndLog(outputHelper);
+        }
+
+        [Fact]
+        public void SortingVisitor_ShuffleTest()
+        {
+            bool areEqual;
+            // create a test sarif log
```

Comment:
>// create a test sarif log [](http://example.com/codeflow?start=12&length=26)

I got a note from Michael in one of my PR that, 
comments should be like a sentence start with Capital and ends with a ending.

But that was on a header of a method, not sure if also applies here?

Reply:
Yes, this note applies. This is not a helpful comment in any case.